sdk/src/store.rs

```diff
@@ -2056,7 +2056,7 @@ impl Store {
                         .validation_status(validation_status::ASSERTION_TIMESTAMP_MALFORMED)
                         .failure_as_err(
                             validation_log,
```

Comment:
This section is a bad merge,  the change made here should be undone.

Reply:
removed